Automattic/jetpack
# Contact Form Block: Fix issues with FSE

Fixes https://github.com/Automattic/jetpack/issues/19462

#### Changes proposed in this Pull Request:

* Fix CSS issues with using the contact form box in the site editor. This PR adds some new base styles for the form the front end. Primarily the padding that was missing.
* Fixes an issue where a textarea would not be 100% in the editor
* Fixes an issue with custom widths of contact form fields

Note: this will not match TwentyTwentyOne theme form styles. These are not present in the blocks version. Future block themes will include some more base form styles. This PR will simply add some base styles so the form is more presentable in the front end.

Before (missing padding etc)

![113967711-e7f2a800-9874-11eb-8273-0f17e493adaf](https://user-images.githubusercontent.com/1464705/114770940-dd2a8e00-9d20-11eb-808a-470e46f2dfa0.png)

After (Includes padding matching editor and correct button styles)

<img width="685" alt="Screen Shot 2021-04-14 at 12 52 16 PM" src="https://user-images.githubusercontent.com/1464705/114771082-064b1e80-9d21-11eb-9987-c0b254204616.png">


#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Enable TT1 blocks theme.
* Insert a contact form block in the site editor.
* Confirm that the contact form fields display as expected in the editor.
* Confirm that the contact form fields display as expected in the front end.
* Switch to a regular WP theme and insert a contact form block in a post.
* Confirm that in this post the contact form fields work expected in editor and front end.
